projects/addon-table/components/table/table.options.ts

```diff
@@ -2,8 +2,14 @@ import type {Provider} from '@angular/core';
 import {tuiCreateToken, tuiProvideOptions} from '@taiga-ui/cdk/utils/miscellaneous';
 import type {TuiSizeL, TuiSizeS} from '@taiga-ui/core/types';
 
+export const TuiDirection = {
```

Comment:
We already have many `Tui*Direction` types/constants: `TuiSwipeDirection`, `TuiGradientDirection`, `TuiHintDirection` etc.
New constant with just single word `direction` can be rather confusing.

Let's rename this const `TuiDirection` => `TuiSortDirection`.